app/services/course_search_service.rb

```diff
@@ -57,4 +58,10 @@ def study_types
 
     filter[:study_type].split(",")
   end
+
+  def provider_name
+    return [] if filter[:"provider.provider_name"].blank?
+
+    filter[:"provider.provider_name"].split(",")
```

Comment:
Are we expecting multiple provider names? 

``` ruby
[1] pry(main)> filter = {"provider.provider_name": "a,b,c"}
=> {:"provider.provider_name"=>"a,b,c"}
[2] pry(main)> filter[:"provider.provider_name"].split(",")    
=> ["a", "b", "c"]
```

Reply:
There's no expectation of it, but it was the easiest way of turning the data into an array! If you know a better way I'd be happy to replace it

Reply:
There are a small number of providers that have commas in their names:

```
> RecruitmentCycle.current.providers.where("provider_name LIKE '%,%'").count
=> 41
```

Are they being escaped properly on the client side?

If all you want is conversion to an Array, then `Array()` works:

```
> Array("foo")
=> ["foo"]
> Array(["foo"])
=> ["foo"]
```

Reply:
Cool, I'll change this then